### ISSUE TITLE
groups: attempting to archive a group results in an in-landscape error

### ISSUE THREAD

**Issue by urcades** [latter-bolden/port#88]
I was trying to archive my group while using taisho, and the button to archive continued to error out on me.

I documented this issue here: https://github.com/urbit/landscape/issues/872

After trying to delete the group in any other browser, it looks like what may be missing is a means of handling this sort of interface:

<img width="536" alt="Screen Shot 2021-05-12 at 10 12 56 AM" src="https://user-images.githubusercontent.com/1195363/117990249-fa944d00-b30a-11eb-879e-7647e7501667.png">

In a nutshell, it looks like taisho will need a way to handle input-style alerts in this manner.

**Comment by arthyn** [comment]
We can either use something easy like in these comments https://stackoverflow.com/questions/38858879/replace-prompt-in-on-electron or implement our own using the `Dialog` component from radix and adding handlers in the main window preload script.
